test/jdk/java/nio/channels/DatagramChannel/SelectWhenRefused.java

```diff
@@ -121,11 +125,12 @@ public static void main(String[] args) throws IOException {
     static void sendDatagram(DatagramChannel dc, SocketAddress remote)
         throws IOException
     {
-        ByteBuffer bb = ByteBuffer.wrap("Greetings!".getBytes());
+        ByteBuffer bb = ByteBuffer.wrap("Greetings from SelectWhenRefused!".getBytes());
```

Comment:
You could put the message in a static string constant, that would avoid repeating it.